lib/openapi/revisions/vwrs-spec-2.0.0-20240521.json

```diff
@@ -3,11 +3,14 @@
   "info": {
```

Comment:
This is a completely different file, not a rename of an existing one.
- I've deleted static/swagger/vwrs-spec.json, because it seems to be (1) out of date and (2) not used.
- I've added the vwrs-spec-2.0.0-20240521.json file, because the instructions page says that it should be added (https://www.notion.so/macrometa/Steps-to-update-PhotonIQ-VWRS-API-reference-documentation-e451f63422c64a1d9e29ee12b4f1c05c)

Reply:
I don't think you even need this.  Just delete it completely.  The documentation above should be changed to not create this anymore.  This goes back to the days we were storing everything in the swagger director.